runelite-client/src/main/java/net/runelite/client/plugins/friendnotes/FriendNotesPlugin.java

```diff
@@ -61,10 +62,13 @@
 public class FriendNotesPlugin extends Plugin
 {
 	private static final String CONFIG_GROUP = "friendNotes";
+	private static final String CONFIG_GROUP_IGNORE = "ignoreNotes";
```

Comment:
It would probably make more sense to reuse the config group for friendnotes, since a player cannot be on both your friend list and ignore list, and it would let the note migrate between lists.